testing/helpers.ts

```diff
@@ -171,12 +171,15 @@ class JHipsterRunContext extends RunContext<GeneratorTestType> {
   /**
    * Lookup generators at generator-jhipster's parent at a npm repository
    * @param lookups generators relative folder
+   * @param packagePath overriden package path for the blueprint
    * @returns
    */
-  withParentBlueprintLookup(lookups = ['generators']): this {
+  withParentBlueprintLookup(lookups = ['generators'], packagePath?: string | [string]): this {
     const packageRootParent = join(getPackageRoot(), '..');
     if (basename(packageRootParent) === 'node_modules') {
       this.withLookups([{ packagePaths: [join(packageRootParent, '..')], lookups }] as any);
+    } else if (packagePath) {
+      this.withLookups([{ packagePaths: Array.isArray(packagePath) ? packagePath : [packagePath], lookups }] as any);
```

Comment:
I don't see a reason for this additional api.
`withLookups({ packagePaths: customPackagePath)`
Is cleaner than:
`withParentBlueprintLookup(undefined, customPackagePath)`

Reply:
yeah agreed, I will accept your other suggestion and remove the param from the function and let process.cwd() do its trick.